packages/victory-core/src/victory-accessibility-group/victory-accessibility-group.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+class VictoryAccessibilityGroup extends React.Component {
```

Comment:
preference of Victory _Accessible_ Group vs _Accessibility_ ?

Reply:
I think `VictoryAccessibleGroup` reads a little nicer, but I don't have a strong preference.